src/main/scala/com/rallyhealth/sbt/versioning/GitVersioningPlugin.scala

```diff
@@ -135,7 +143,7 @@ object GitVersioningPlugin extends AutoPlugin {
     autoFetchTimeout := 15, // seconds
 
     autoFetchResult := {
-      implicit val logger: Logger = ConsoleLogger()
+      implicit val logger: Logger = sLog.value
```

Comment:
This is different from 1.9.0, but I think it makes more sense to inject the logger based on settings, rather than construct it here.